test/either-spec.js

```diff
@@ -92,6 +92,16 @@ describe('An Either', function () {
         it('renders as Right(value)', function() {
           expect(rightString.toString()).toBe('Right(abcd)')
         })
+        it('will execute side-effects on forEach', function() {
+          var t = []
+          rightString.forEach(function (x) {t.push(x)})
+          expect(t).toEqual(['abcd'])
```

Comment:
better use Jasmine spies here --> https://jasmine.github.io/2.4/introduction#section-Spies

and in all other side effects tests ;)